src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -1148,4 +1156,10 @@ export default class DefaultAudioVideoController
       );
     });
   }
+
+  private getScaleDownFactorFromUplinkPolicy(): number {
+    return this.meetingSessionContext.videoUplinkBandwidthPolicy.scaleResolutionDownBy
+      ? this.meetingSessionContext.videoUplinkBandwidthPolicy.scaleResolutionDownBy()
+      : 1;
+  }
```

Comment:
This seems like we're going in the wrong direction for designing the interface for policies, no? Should policies be in charge of producing the `encodeParam` value or something more central, rather than returning a bunch of these individual fields that then get threaded through to the transceiver controller?

Reply:
I agrees that it is confusing as defaultAudioVideoController actually tries to set these values and pass them through the transceiver, it is our existing code to handle the active speaker case so I would like to keep it that way to reduce the scope of this PR.

Reply:
Updated.